src/Microsoft.DotNet.Build.Tasks/Targets/resources.targets

```diff
@@ -23,16 +21,35 @@
           Inputs="$(StringResourcesPath)"
           Outputs="$(IntermediateResOutputFileFullPath)">
     
-    <GenerateResourcesCode 
+    <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
-        IntermediateFilePath="$(IntermediateResOutputFileFullPath)" 
-        OutputSourceFilePath="$(ResourceSRPath)" 
+        OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
         AssemblyName="$(AssemblyName)" />
 
     <ItemGroup>
+      <!-- The following Compile element has to be included dynamically inside the Target otherwise intellisense will not work -->
+      <Compile Include="$(IntermediateResOutputFileFullPath)">
+          <Visible>true</Visible>
```

Comment:
I think you are right. I re-checked that and is not visible. I'll remove <Visible> element